psi4/src/read_options.cc

```diff
@@ -1544,6 +1544,30 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- Bump function max radius -*/
         options.add_double("DF_BUMP_R1", 0.0);
 
+        /*- SUBSECTION COSX Algorithm -*/
+
+        /*- Number of spherical points in initial COSX grid. -*/
+        options.add_int("COSX_SPHERICAL_POINTS", 50);
+        /*- Number of radial points in initial COSX grid. -*/
+        options.add_int("COSX_RADIAL_POINTS", 25);
```

Comment:
It may be worth considering renaming the initial grid variables to COSX_SPHERICAL_POINTS_INITIAL and COSX_RADIAL_POINTS_INITIAL, as this clarifies their role in the calculation and makes the initial grid keywords match more closely with the final grid keywords.

Reply:
Yeah, fair enough